plugins/orchestrator/src/utils/formState.ts

```diff
@@ -0,0 +1,35 @@
+import { JsonObject, JsonValue } from '@backstage/types';
+
+export function isJsonObject(value: JsonValue): value is JsonObject {
+  return typeof value === 'object' && value !== null && !Array.isArray(value);
+}
+
+export function flattenParametersFromFormState(
+  formState?: Record<string, JsonValue>,
+): Record<string, JsonValue> {
+  if (!formState) {
+    return {};
+  }
+
+  const parameters: Record<string, JsonValue> = {};
+
+  Object.entries(formState).forEach(([key, value]) => {
+    if (value === undefined) {
+      parameters[key] = '';
```

Comment:
```suggestion
      parameters[key] = null;
```

null is a valid JsonValue, seems like the more correct value to put here